darshan-util/pydarshan/darshan/backend/cffi_backend.py

```diff
@@ -300,13 +300,15 @@ def log_get_record(log, mod, dtype='numpy'):
 
 
 
-def log_get_generic_record(log, mod_name, dtype='numpy'):
+def log_get_generic_record(log, mod_name, c_cols=None, fc_cols=None, dtype='numpy'):
```

Comment:
A bit unfortunate to have to change the call signature--at the very least, you should put the new arguments at the end of the list so that any examples that previously only used values (without keywords) don't break.

For example, if there was a notebook somewhere that previously did:

`log_get_generic_record(log_handle, "POSIX", "pandas")`

this change is now guaranteed to break that, becuase `c_cols=pandas` doesn't make sense

Reply:
That's a good point. I'll make sure to fix that.

Reply:
This can be resolved now that the API changes have been removed.